asylum-connect/control-panel
# Dev1198892993658133/improve mark verified

## Description



- Asana ticket: 1198892993658133

## PR Checklist



- [X] Assign @FJKhan as a reviewer.

## How to Test

Login to Control Panel
Select an organization
Click 'Edit Details' button
Edit necessary details
Click new button 'Confirm and Verify'
'Verify information for _OrgName_?' appears in verify modal.
Cancel or Confirm the verification.
Edits are saved, Last verified timestamp is updated on success.
